src/lib/output/themes/default/partials/header.tsx

```diff
@@ -8,7 +8,9 @@ export const header = (context: DefaultThemeRenderContext, props: PageEvent<Refl
     const HeadingLevel = props.model.isProject() ? "h2" : "h1";
     return (
         <div class="tsd-page-title">
-            {!!props.model.parent && <ul class="tsd-breadcrumb">{context.breadcrumb(props.model)}</ul>}
+            {!!props.url && props.url !== "index.html" && props.url !== "hierarchy.html" && (
```

Comment:
`props.url !== "index.html"` is the only branch here that's possible, there will always be a url at this point, and this helper isn't called for the hierarchy page

Reply:
Just as a test I added the following code before the return and it does trigger, so `header.tsx` is used for `hierarchy.html`.

```ts
if (props.url === "hierarchy.html") {
    console.log("hierarchy matched");
}
```

I do agree that the `!!props.url && ` is a bit useless on a second glance though.

Reply:
Huh, I apparently got very confused when looking at this, LGTM!